lib/awful/widget/prompt.lua

```diff
@@ -57,7 +57,7 @@ end
 --
 -- @param args Arguments table. "prompt" is the prompt to use.
 -- @return A launcher widget.
--- @name awful.widget.prompt
+-- @function awful.prompt
```

Comment:
I'm only sure that what is currently there is wrong:

/usr/src/awesome-build/lib/awful/widget/prompt.lua:69: no type specified, assuming function: 'awful.widget.prompt'